openmls/src/framing/mod.rs

```diff
@@ -13,15 +48,21 @@ use crate::schedule::{message_secrets::*, *};
 use serde::{Deserialize, Serialize};
 use tls_codec::*;
 
-pub(crate) mod ciphertext;
 pub(crate) mod codec;
 pub(crate) mod message;
-pub(crate) mod plaintext;
+pub(crate) mod mls_auth_content;
+pub(crate) mod mls_content;
+pub(crate) mod private_message;
+pub(crate) mod public_message;
 pub(crate) mod sender;
 pub(crate) mod validation;
-pub(crate) use ciphertext::*;
 pub(crate) use errors::*;
-pub(crate) use plaintext::*;
+pub(crate) use mls_auth_content::*;
```

Comment:
I see unused import warnings for this now. I think you still important things from `mls_auth_content::` manually.